src/WebJobs.Script.WebHost/Security/Authorization/AuthUtility.cs

```diff
@@ -42,6 +42,10 @@ public static bool PrincipalHasAuthLevelClaim(ClaimsPrincipal principal, Authori
                 if (level == AuthorizationLevel.Admin)
                 {
                     return true;
+                } else if (isDisabled)
+                {
+                    // If the identity is not authenticated with Admin level and function is disabled
```

Comment:
Not sure the comment is necessary. With the changed parameter name, this will be clear.